packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -470,59 +457,117 @@ export default class DatePicker extends React.Component {
-      dayClassName={this.props.dayClassName}
-      showTimeSelect={this.props.showTimeSelect}
-      onTimeChange={this.handleTimeChange}
-      timeFormat={this.props.timeFormat}
-      timeIntervals={this.props.timeIntervals}
-      minTime={this.props.minTime}
-      maxTime={this.props.maxTime}
-      excludeTimes={this.props.excludeTimes}
-      className={this.props.calendarClassName}
-      yearDropdownItemNumber={this.props.yearDropdownItemNumber}>
-      {this.props.children}
-    </WrappedCalendar>
+
+    if (this.props.withPortal) {
+      return <WrappedCalendar
+        ref={(elem) => { this.calendar = elem }}
+        locale={this.props.locale}
+        adjustDateOnChange={this.props.adjustDateOnChange}
+        setOpen={this.setOpen}
+        dateFormat={this.props.dateFormatCalendar}
+        useWeekdaysShort={this.props.useWeekdaysShort}
+        dropdownMode={this.props.dropdownMode}
+        selected={this.props.selected}
+        preSelection={this.state.preSelection}
+        onSelect={this.handleSelect}
+        onWeekSelect={this.props.onWeekSelect}
+        openToDate={this.props.openToDate}
+        minDate={this.props.minDate}
+        maxDate={this.props.maxDate}
+        selectsStart={this.props.selectsStart}
+        selectsEnd={this.props.selectsEnd}
+        startDate={this.props.startDate}
+        endDate={this.props.endDate}
+        excludeDates={this.props.excludeDates}
+        filterDate={this.props.filterDate}
+        onClickOutside={this.handleCalendarClickOutside}
+        formatWeekNumber={this.props.formatWeekNumber}
+        highlightDates={this.state.highlightDates}
+        includeDates={this.props.includeDates}
+        inline={this.props.inline}
+        peekNextMonth={this.props.peekNextMonth}
+        showMonthDropdown={this.props.showMonthDropdown}
+        showWeekNumbers={this.props.showWeekNumbers}
+        showYearDropdown={this.props.showYearDropdown}
+        withPortal={this.props.withPortal}
+        forceShowMonthNavigation={this.props.forceShowMonthNavigation}
+        scrollableYearDropdown={this.props.scrollableYearDropdown}
+        todayButton={this.props.todayButton}
+        weekLabel={this.props.weekLabel}
+        utcOffset={this.props.utcOffset}
+        outsideClickIgnoreClass={outsideClickIgnoreClass}
+        fixedHeight={this.props.fixedHeight}
+        monthsShown={this.props.monthsShown}
+        onDropdownFocus={this.handleDropdownFocus}
+        onMonthChange={this.props.onMonthChange}
+        dayClassName={this.props.dayClassName}
+        showTimeSelect={this.props.showTimeSelect}
+        onTimeChange={this.handleTimeChange}
+        timeFormat={this.props.timeFormat}
+        timeIntervals={this.props.timeIntervals}
+        minTime={this.props.minTime}
+        maxTime={this.props.maxTime}
+        excludeTimes={this.props.excludeTimes}
+        className={this.props.calendarClassName}
+        yearDropdownItemNumber={this.props.yearDropdownItemNumber}>
```

Comment:
If the `>` was moved to it's own line it would be a lot easier to tell that `{this.props.children}` is a child and not another prop. Similar to how the `Calendar` below is formatted.

Reply:
Fixed